TestChangelogSort is broken

### Prerequisites

* [x] Are you running the latest version?
* [x] Did you perform a search to see if this problem was already reported?

For more information, see the `CONTRIBUTING` guide.

### Description

```
--- FAIL: TestChangelogSort (0.08s)
    --- FAIL: TestChangelogSort/changelog_sort='' (0.01s)
        Error Trace:    changelog_test.go:124
    	Error:		Not equal: []string{"b: commit", "a: commit", "c: commit"} (expected)
    			        != []string{"(HEAD -> master, tag: v1.0.0) b: commit", "a: commit", "c: commit"} (actual)

    --- FAIL: TestChangelogSort/changelog_sort='asc' (0.01s)
        Error Trace:    changelog_test.go:124
    	Error:		Not equal: []string{"a: commit", "b: commit", "c: commit"} (expected)
    			        != []string{"(HEAD -> master, tag: v1.0.0) b: commit", "a: commit", "c: commit"} (actual)

    --- FAIL: TestChangelogSort/changelog_sort='desc' (0.01s)
        Error Trace:    changelog_test.go:124
    	Error:		Not equal: []string{"c: commit", "b: commit", "a: commit"} (expected)
    			        != []string{"c: commit", "a: commit", "(HEAD -> master, tag: v1.0.0) b: commit"} (actual)
```

`git bisect` returns this as the first bad commit

```
6277ab9ae3674188863371e3a75267ef05b767df is the first bad commit
commit 6277ab9ae3674188863371e3a75267ef05b767df
Author: Carlos Alexandro Becker <caarlos0@gmail.com>
Date:   Thu Oct 19 20:44:12 2017 -0200

    feat: added sorting support to the changelog pipe

    We can now add:

    ```yaml
    changelog:
            sort: asc
    ```

    to group together our commits by prefix or just to sort them...

    Closes #284

:040000 040000 29eccd67829d64bf1b1b70433ac2991438f9c5f2 eedbbf6912b4d20050e76cb236fb7b7ba97c3fc3 M	config
:040000 040000 b2b2ee539b5bea26d2ca7dd83e98dc3f52299f5f 95073bc67986d4b7cd7a626868d6e5b01fff4343 M	docs
:040000 040000 8618bb95ccf520f8a3c05ccdd80b4e493eebfb13 793b609ac11f2ddb474402d27c4a246c59b42c5e M	pipeline
```

### Steps to Reproduce

```
go test ./pipeline/changelog/
```

--- a/pipeline/changelog/changelog.go
+++ b/pipeline/changelog/changelog.go
@@ -124,7 +124,7 @@ func getChangelog(tag string) (string, error) {
 }
 
 func gitLog(refs ...string) (string, error) {
-	var args = []string{"log", "--pretty=oneline", "--abbrev-commit"}
+	var args = []string{"log", "--pretty=oneline", "--abbrev-commit", "--no-decorate"}
 	args = append(args, refs...)
 	return git.Run(args...)
 }

